Modules/CustomRolesHelper.cs

```diff
@@ -260,6 +260,7 @@ public static RoleTypes GetDYRole(this CustomRoles role) // Role has a kill butt
             CustomRoles.Agitater => RoleTypes.Impostor,
             CustomRoles.Spiritcaller => RoleTypes.Impostor,
             CustomRoles.ChiefOfPolice => RoleTypes.Impostor,
+            CustomRoles.Quizmaster => RoleTypes.Impostor,
```

Comment:
This function is used to make desync impostor roles (neutrals and crewmates with impostor basis) and you don't need to add normal impostor roles to this